### ISSUE TITLE
Conformance claim needs feedback mechanisms

### ISSUE THREAD

**Issue by jenoe** [w3c/silver#304]
## Reasoning

Learning from users' experiences and feedback is an inherent aspect of accessibility.

## Proposal

Extend the required components of a conformance claim by at **least one accessible feedback mechanism**, eg. an email address, a web form.

## Reference

The [EU directive on the accessibility of the websites and mobile applications of public sector bodies](https://eur-lex.europa.eu/LexUriServ/LexUriServ.do?uri=CELEX:32016L2102:EN:HTML#d1e40-1-1) requires such a feedback mechanism for every accessibility statement.

**Comment by LJWatson** [comment]
I agree that having a feedback mechanism in a conformance claim is a good idea, but caution against referencing the EU Directive. A WCAG3.0 conformance claim is not compatible with an accessibility statement under that Directive.

**Comment by jenoe** [comment]
> ..., but caution against referencing the EU Directive. A WCAG3.0 conformance claim is not compatible with an accessibility statement under that Directive.

I agree. It was just meant to be an example.

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by chaals** [comment]
A cautious +1 to the proposal.

Some possible drawbacks are that it could produce a vector for denial-of-service attacks, flooding the feedback system with garbage to the point that it is not feasible to continue using it properly, and that fear of such an event could lead some to decide to avoid the whole thing - and others to use it as an excuse for inaction. It is also likely that content, including accessibility statements, will survive longer than the capability to use the feedback.

Recognising these issues, I still think it is worth proceeding with the proposal.

**Comment by MakotoUeki** [comment]
+1 to @chaals. I think It's worth considering as well.

In Japan, there used to be a requirement in the JIS (Japanese standard) to require a website to provide an accessible mechanisms to get feedback from users. JIS doesn't have it any more because the JIS was updated to become identical to ISO/IEC 40500:2012 (= WCAG 2.0) in 2016.

**Comment by SuzanneTaylor** [comment]
Thank you for this comment, we discussed this in the Silver Friday meetings on November 12 and November 13. The group agreed that this is important and we felt that the best place for this to live would be within the Maturity Model, where detailed feedback about ensuring the quality of the feedback mechanism could accompany it. 

This was already in the Communications section of the Maturity Model and, in response to this comment, the group working on the Maturity Model also added it to the Support / Development Life Cycle Section, so that we can help guide meaningful handling of the feedback.

You can see the draft in Google docs here: 

- https://docs.google.com/document/d/1Y5EO6zkOMrbyePw5-Crq8ojmhn9OCTRQ6TlgB0cE6YE/edit#heading=h.qi5v5i5fuzz0

It will also be added to the Github version here:

- https://github.com/w3c/silver/issues/304
